snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml

```diff
@@ -8,6 +8,50 @@ device:
 
 sysobjectid: 1.3.6.1.4.1.318.1.*
 
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "apc"
+      serial_number:
+        symbol:
+          MIB: POWERNET-MIB
+          OID: 1.3.6.1.4.1.318.1.1.1.1.2.3.0
+          name: upsAdvIdentSerialNumber
+          # The serial number of the UPS internal microprocessor.  This number is set at
+          # the factory. NOTE: This number does NOT correspond to the serial number on the rear of the UPS.
+          # Examples: NA0835009156, AQ1805133700
+      version:
+        symbol:
+          OID: 1.3.6.1.4.1.318.1.1.1.1.2.1.0
+          name: upsAdvIdentFirmwareRevision
+      product_name:
+        symbol:
+          OID: 1.3.6.1.4.1.318.1.1.1.1.1.1.0
+          name: upsBasicIdentModel
+      model:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          match_pattern: 'MN:(\w+)'
+          match_value: '$1'
+          # Examples: MN:AP9631
+      os_name:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          match_pattern: 'PN:apc_hw(\d+)_aos_'
+          match_value: 'AOS'
```

Comment:
Is it possible to have something else than AOS?

Reply:
For now, it seems not.

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What does `match_value: 'AOS'` stands for? What would be the os_name from the given example ?

Reply:
Does this mean that `os_name` will be `AOS` if pattern `PN:apc_hw(\d+)_aos_` is matched ?

Reply:
In `match_value`, we can also, use the captured groups like this:

```
          match_pattern: 'PN:apc_hw(\d+)_aos_'
          match_value: '$1'
```

That would result in `$1` being whatever is capture by `(\d+)`

In the case of:

```
          match_pattern: 'PN:apc_hw(\d+)_aos_'
          match_value: 'AOS'
```

We don't care what is capture (we don't use $1, $2, etc) and just want to use the value `AOS` whenever  `PN:apc_hw(\d+)_aos_` can match the sysDescr.

Reply:
So it means that os_name will be `AOS` if pattern `PN:apc_hw(\d+)_aos_` is matched ?

Reply:
> So it means that os_name will be AOS if pattern PN:apc_hw(\d+)_aos_ is matched ?

Yes